dev-games/tiled/tiled-1.11.0.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils
+
+DESCRIPTION="A general purpose tile map editor"
+HOMEPAGE="https://www.mapeditor.org/"
+SRC_URI="https://github.com/mapeditor/tiled/archive/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD BSD-2 GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="clang examples"
+
+RDEPEND="
+	app-arch/zstd
+	dev-qt/qtbase:6[X]
+	dev-qt/qtimageformats:6
+	dev-qt/qtdeclarative:6
+	dev-qt/qtwayland:6
+	sys-libs/zlib
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	dev-util/qbs
+"
+
+src_configure() {
+	qbs setup-qt /usr/bin/qmake6 qt6 || die
+	qbs config defaultProfile qt6 || die
+
+	if use clang; then
```

Comment:
Is a use flag required versus using `qbs setup-toolchains --detect` to use the users CC and CXX?

If so use of tc-is-clang from toolchain-funcs to detect what the user wants is also an option.

Reply:
Most LLVM setups (including Gentoo LLVM profile) also have GCC installed to build packages failing to compile with clang. If both are installed, `qbs setup-toolchains --detect` will add profiles for both and GCC profile will be chosen as default, causing compilation error as Qt was built with clang and libcxx.